src/App.vue

```diff
@@ -1,8 +1,11 @@
 <script setup lang="ts">
-import TheNavbar from "./components/TheNavbar.vue";
-import TheDrawer from "./components/TheDrawer.vue";</script>
+// import TheNavbar from "./components/TheNavbar.vue";
```

Comment:
Why are these commented? Components are used in templates.

Reply:
Was commented in streams code, uncommented